infrastructure/src/main/java/org/corfudb/infrastructure/configuration/CorfuConfig.java

```diff
@@ -0,0 +1,20 @@
+package org.corfudb.infrastructure.configuration;
+
+import lombok.Getter;
+import org.corfudb.infrastructure.ServerContext;
+
+import java.util.Map;
+
+/**
+ * Global configuration for Corfu.
+ */
+@Getter
+public class CorfuConfig {
+    private final Map<String, Object> config;
```

Comment:
Why having another reference here besides the one already in ServerContext? Can we just use SeverContext and if you need `config`, just add a Getter() on it?

Reply:
Fixed